fe/fe-core/src/main/java/org/apache/doris/qe/SimpleScheduler.java

```diff
@@ -92,6 +92,38 @@ public static TNetworkAddress getHost(long backendId,
                         backends, locations.size()));
     }
 
+    public static TScanRangeLocation getLocation(TScanRangeLocation minLocation,
```

Comment:
Why add this method?  The diff between `getLocation` and `getHost` is first arg, but we could get `long backendId` from `TScanRangeLocation minLocation`?

Reply:
Because function `computeScanRangeAssignmentByScheduler` need `minLocation` to set volume_id. Therefore it must return param `minLocation`

// Volume is optional, so we need to set the value and the is-set bit
scanRangeParams.setVolumeId(minLocation.volume_id);